src/chat/components/CustomBubble.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useRef, useState } from 'react';
+import {
+  StyleSheet,
+  View,
+  Pressable,
+  Image,
+  TouchableOpacity,
+} from 'react-native';
+import Video, { VideoRef } from 'react-native-video';
+import FastImage from 'react-native-fast-image';
+import type { MessageProps } from 'src/interfaces';
+
+interface CustomBubbleProps {
+  message: MessageProps;
+  position: 'left' | 'right';
+  selectedImgVideoUrl: (url: string) => void;
+}
+
+export const CustomBubble: React.FC<CustomBubbleProps> = ({
+  position,
+  message,
+  selectedImgVideoUrl,
+}) => {
+  const [isPauseVideo, setIsPauseVideo] = useState(true);
+  const videoRefs = useRef<VideoRef>(null);
+
+  return (
+    <View
+      style={[
+        styles.bubleContainer,
+        position === 'left' ? styles.flexStart : styles.flexEnd,
+      ]}
+    >
+      <Pressable
+        onPress={() =>
+          message.path &&
+          message.type === 'photo' &&
+          selectedImgVideoUrl(message.path)
+        }
+        style={styles.bubble}
+      >
+        {message.type === 'photo' ? (
```

Comment:
The type of message is photo, video and text. Where is the logic for "text"?

Reply:
the text logic used default Bubble. I already changed name to **CustomImageVideoBubble** more easy to read